index.html

```diff
@@ -13,39 +13,254 @@
+              </li>
+            </ul>
+          </div>
+          <button class="button">Get my resume</button>
+          <div class="languages">
+            <ul>
+              <li class="language-item">
+                <h1>Languages</h1>
+                <a href="#"><svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path fill-rule="evenodd" clip-rule="evenodd"
+                    d="M5.29289 8.29289C5.68342 7.90237 6.31658 7.90237 6.70711 8.29289L12 13.5858L17.2929 8.29289C17.6834 7.90237 18.3166 7.90237 18.7071 8.29289C19.0976 8.68342 19.0976 9.31658 18.7071 9.70711L12.7071 15.7071C12.3166 16.0976 11.6834 16.0976 11.2929 15.7071L5.29289 9.70711C4.90237 9.31658 4.90237 8.68342 5.29289 8.29289Z"
+                    fill="#5E6C84" />
+                </svg></a>
+              </li>
+              <div class="language language-body">
+                <div class="language-icon">
+                  <img src="./Images/Ellipse 1.png" alt="">
+                </div>
+                <span>JavaScript</span>
+              </div>
+              <div class="language language-body">
+                <div class="language-icon">
+                  <img src="./Images/Ellipse_2.png" alt="">
+                </div>
+                <span>HTML</span>
+              </div>
+              <div class="language language-body">
+                <div class="language-icon">
+                  <img src="./Images/Ellipse_3.png" alt="">
+                </div>
+                <span>CSS</span>
+              </div>
+              <li class="language-item framework-skills">
+                <h1>Frameworks</h1>
+                <a href="#"><svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path fill-rule="evenodd" clip-rule="evenodd"
+                    d="M9.29289 5.43962C9.68342 5.0491 10.3166 5.0491 10.7071 5.43962L16.7071 11.4396C17.0976 11.8301 17.0976 12.4633 16.7071 12.8538L10.7071 18.8538C10.3166 19.2444 9.68342 19.2444 9.29289 18.8538C8.90237 18.4633 8.90237 17.8301 9.29289 17.4396L14.5858 12.1467L9.29289 6.85384C8.90237 6.46331 8.90237 5.83015 9.29289 5.43962Z"
+                    fill="#505F79" />
+                </svg></a>
+              </li>
+              <li class="language-item framework-skills">
+                <h1>Skills</h1>
+                <a href=""><i class="fa-solid fa-angle-right"></i></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+  </section>
+  <section class="contact-me">
+      <div class="contact-header">
+        <h2 class="contact-title">Contact Me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that
+          needs
+          coding. I’d love to help with it</p>
+      </div>
+      <form class="contact-form" action="https://formspree.io/html/" method="post">

```

Comment:
- [ ] KIndly make sure your `action` value is similar to `https://formspree.io/f/{form_id}`. To generate your unique `form_id`, please create an account on [formspree service](https://formspree.io/html) and copy the generated `form_id` into action after that the user will be able to submit the form successfully. :wink: